wct-core/src/test/java/org/webcurator/core/profiles/Heritrix3ProfileTest.cxml

```diff
@@ -34,7 +34,7 @@
  <bean id="simpleOverrides" class="org.springframework.beans.factory.config.PropertyOverrideConfigurer">
   <property name="properties">
    <value>
-metadata.operatorContactUrl=http://www.natlib.govt.nz/
+metadata.operatorContactUrl=http://www.national-library-archive.com/
```

Comment:
This isn't a valid URL, but it's not obvious that this is just an example URL.

Reply:
Updated to an obvious test url "http://www.test-contact-url.com/"